livy/src/main/java/org/apache/zeppelin/livy/LivySparkSQLInterpreter.java

```diff
@@ -141,6 +171,10 @@ public InterpreterResult interpret(String line, InterpreterContext context) {
       if (result.code() == InterpreterResult.Code.SUCCESS) {
         InterpreterResult result2 = new InterpreterResult(InterpreterResult.Code.SUCCESS);
         for (InterpreterResultMessage message : result.message()) {
+          // when sql content is "use xxx", message.getData() is "df: org.apache.spark.sql.DataFrame = []"
+          if (message.getType() == InterpreterResult.Type.TEXT && message.getData().equals("df: org.apache.spark.sql.DataFrame = []")){
```

Comment:
Why ignore this kind of message? What is the output if you run sql statements in spark-sql client side?

Reply:
when sql content is DDL and tableWithUTFCharacter, message.getData() is:
```
df: org.apache.spark.sql.DataFrame = []
```
which will lead to OutOfIndexException at [here](https://github.com/apache/zeppelin/blob/d2cdb77f6ae6a13ff9c770b82d7125270415a77c/livy/src/main/java/org/apache/zeppelin/livy/LivySparkSQLInterpreter.java?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L183)



when sql content is DDL and tableWithUTFCharacter, message.getData() is:
```
++
||
++
++
```
which means an empty table.

Reply:
I have made a change in the latest submit, func `parseSQLJsonOutput` will return empty List (means empty table without header) when result message is less than 2 lines, and **all empty table without header** response will be replaced by text `success`.